sdk/core/azure-core/inc/azure/core/io/body_stream.hpp

```diff
@@ -220,7 +219,7 @@ namespace Azure { namespace Core { namespace IO {
        * for the necessary duration. The caller is also responsible for closing it once they are
        * done.
        */
-      RandomAccessFileBodyStream(HANDLE fileHandle, int64_t offset, int64_t length)
+      RandomAccessFileBodyStream(void* fileHandle, int64_t offset, int64_t length)
```

Comment:
Not a fan of `void*` here :(

Reply:
maybe we don't avoid including windows.h here.
If you use `RandomAccessFileBodyStream`, you nave to have it included.
So maybe either be fine with including windows.h, or extract `RandomAccessFileBodyStream` to a separate header - so that if you don't use it, Win32 does not pop in your intellisense.

Reply:
This type is internal, so `void*` doesn't bother me as much. Not removing `windows.h` here in `body_stream.hpp` would effectively mean all SDK users get exposed to it, which is precisely what we want to avoid, as part of this.

Reply:
I see. If for internal - I don't care about `void*` too much :)
Maybe it is better to move internal stuff to a header in `internal/`.